SDK/MobileCenterCrashes/Microsoft.Azure.Mobile.Crashes.Shared/Crashes.cs

```diff
@@ -1,13 +1,79 @@
 ﻿using System;
-using Microsoft.Azure.Mobile.Crashes.Shared;
 
 namespace Microsoft.Azure.Mobile.Crashes
 {
     /// <summary>
-    /// Crashes feature.
+    /// Crashes service.
     /// </summary>
     public static class Crashes
     {
+        static Crashes()
+        {
+            PlatformCrashes.SendingErrorReport += (sender, e) =>
+            {
+                if (SendingErrorReport != null)
+                {
+                    SendingErrorReport(sender, e);
+                }
+            };
+
+            PlatformCrashes.SentErrorReport += (sender, e) =>
+            {
+                if (SentErrorReport != null)
+                {
+                    SentErrorReport(sender, e);
+                }
+            };
+
+            PlatformCrashes.FailedToSendErrorReport += (sender, e) => 
+            {
+                if (FailedToSendErrorReport != null)
+                {
+                    FailedToSendErrorReport(sender, e);
+                }
+            };
+
+            PlatformCrashes.ShouldProcessErrorReport = null;
+            PlatformCrashes.GetErrorAttachment = null;
+        }
+
+        /// <summary>
+        /// Occurs when an error report is about to be sent.
+        /// </summary>
+        public static event SendingErrorReportEventHandler SendingErrorReport;
+
+        /// <summary>
+        /// Occurs when an error report has been successfully sent.
+        /// </summary>
+        public static event SentErrorReportEventHandler SentErrorReport;
+
+        /// <summary>
+        /// Occurs when an error report has failed to be sent.
+        /// </summary>
+        public static event FailedToSendErrorReportEventHandler FailedToSendErrorReport;
+
+        /// <summary>
+        /// Set this callback to add custom behavior for determining whether an error report should be processed.
```

Comment:
Somewhere where its visible ;) Could use remarks section, even it will be visible only in html doc.